<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCavity</class>
 <widget class="QDialog" name="AddCavity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Add Optical Cavity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_9">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="textinput_name">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="button_load">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Energy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="textinput_energy">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="textinput_transitions">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>List of Transitions</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Scaling&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="textinput_transition_scalings">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>List of Scalings</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="button_setone">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Set 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_13">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Decay Scaling&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="textinput_decay">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Photon Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="textinput_photonnumber">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Maximum Photons</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="button_confirm">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="button_reset">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="button_confirm_replace">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Overwrite Original</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="button_coupled_to">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Coupled To</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_2</zorder>
  <zorder>textinput_name</zorder>
  <zorder>label_9</zorder>
  <zorder>textinput_energy</zorder>
  <zorder>label_10</zorder>
  <zorder>textinput_transitions</zorder>
  <zorder>button_confirm</zorder>
  <zorder>button_reset</zorder>
  <zorder>textinput_transition_scalings</zorder>
  <zorder>label_13</zorder>
  <zorder>textinput_decay</zorder>
  <zorder>label_12</zorder>
  <zorder>label_14</zorder>
  <zorder>textinput_photonnumber</zorder>
  <zorder>button_setone</zorder>
  <zorder>button_load</zorder>
  <zorder>button_confirm_replace</zorder>
  <zorder>button_coupled_to</zorder>
 </widget>
 <tabstops>
  <tabstop>textinput_name</tabstop>
  <tabstop>textinput_energy</tabstop>
  <tabstop>textinput_transitions</tabstop>
  <tabstop>textinput_transition_scalings</tabstop>
  <tabstop>textinput_decay</tabstop>
  <tabstop>textinput_photonnumber</tabstop>
  <tabstop>button_confirm</tabstop>
  <tabstop>button_load</tabstop>
  <tabstop>button_setone</tabstop>
  <tabstop>button_confirm_replace</tabstop>
  <tabstop>button_reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
